src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/scanContext.hpp

```diff
@@ -316,6 +320,29 @@ struct TScanContext final
                         m_operationType = OperationType::DeleteAll;
                     }
                 }
+                else if constexpr (std::is_same_v<T, std::shared_ptr<nlohmann::json>>)
+                {
+                    m_data = arg;
+                    auto message = std::get<std::shared_ptr<nlohmann::json>>(m_data);
+                    std::string updateValue = message->value("update", "");
+
+                    if (!updateValue.empty() && updateValue == "success")
+                    {
+                        m_messageType = MessageType::DatabaseTrigger;
+                        m_type = ScannerType::DatabaseUpdate;
+                        m_operationType = OperationType::Insert;
+                    }
```

Comment:
We need to parse the message that Wazuh-DB will send and identify the source and destination upgrade schema.
Not all version upgrades will require the full scan.

Reply:
We need to define this with @Dwordcito